src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -235,40 +235,29 @@ private void setupKeyboardShortcuts() {
                 KeyPress.setLastKeyPressedCodeAndTime(event.getCode());
             }
             if (NEW_COMMENT.match(event)) {
-                if (KeyPress.isValidKeyCombination(GOTO_MODIFIER.getCode(), event.getCode())) {
-                    ui.getBrowserComponent().switchToTab(DISCUSSION_TAB);
-                } else {
-                    if (!ui.getBrowserComponent().isCurrentUrlIssue()) {
-                        openPageOfCurrentlySelectedIssue();
-                    } else if (!ui.getBrowserComponent().isCurrentUrlDiscussion()) {
-                        ui.getBrowserComponent().switchToTab(DISCUSSION_TAB);
+                if (!ui.getBrowserComponent().isCurrentUrlIssue()) {
+                    return;
```

Comment:
Just follow the original implementation, but change any switching tab to open tab instead (of course, do pre-check if necessary to open tab or whatsoever).

The behaviour of the original is:
- if user press `G`+`R`, it will open discussion tab (if on PR, but since if we are on issue switch to tab won't work, there is no guard clause in the original implementation)
- else (meaning user only press `R`), we should jump to comment box. The original implementation have `if (!ui.getBrowserComponent().isCurrentUrlIssue())` to allow user to go to reply box even if they are on non-issue page, e.g. after they selected an issue, they press `G`+`K` which will open the keyboard shortcut page. We expect pressing `R` will jump to the selected issue page's comment box.